_data/people/masonproffitt.yml

```
@@ -49,6 +50,18 @@ presentations:
   location: Princeton University
   focus-area: as
   project: func-adl
+- title: "Data Selection & Delivery for Analysis: ServiceX & funcADL"
+  date: 2020-10-26
+  url: https://indico.cern.ch/event/960587/contributions/4070320/attachments/2129951/3586807/2020-10-26%20ServiceX%20%26%20funcADL.pdf
```

```suggestion
  url: https://indico.cern.ch/event/960587/contributions/4070320/
```

Can you link to the event rather than the PDF? If you ever reuploaded something to that talk this URL would break.

---

It's fine with me, but the [instructions page](https://github.com/iris-hep/iris-hep.github.io-source/blob/804a68277cdc750f43d92ab65f27a9edf36a52a0/pages/docs/newteammember.md?plain=1#L76) should really be updated if this is what's always supposed to be done